bases/renku_data_services/data_api/main.py

```diff
@@ -66,6 +66,15 @@ async def setup_sentry(_):
         app.signal("http.lifecycle.request")(_hub_enter)
         app.signal("http.lifecycle.response")(_hub_exit)
         app.signal("http.routing.after")(_set_transaction)
+    if config.trusted_proxies.forwarded_secret:
+        app.config.FORWARDED_SECRET = config.trusted_proxies.forwarded_secret
+        logger.info(f"FORWARDED_SECRET = {app.config.FORWARDED_SECRET}")
```

Comment:
Lets not print the secret in the logs. You can replace it with just a message that a secret was found and will be used but don't print the secret.

Reply:
I will remove this setting for now.

Reply:
Done in fd7a7abbda27c0c6e6b44b83bc999fe92ec4fe9c.